### ISSUE TITLE
Wrong FS error while training NerCrfApproach

### ISSUE THREAD

**Issue by bairathirahul** [JohnSnowLabs/spark-nlp#283]
I am facing two issues with the version 1.7.0 while training NerCrfApproach
1. I am not able to provide Embeddings from HDFS. I was able to do it with version 1.6.3
2. Now that I have provided the source files from the local file system, I am receiving following error while training.

```
py4j.protocol.Py4JJavaError: An error occurred while calling o858.fit.
: java.lang.IllegalArgumentException: Wrong FS: hdfs:/Users/bairathirahul/hadoop/hdfs/tmp/embeddings992fc2a603c4_idx2045865221115508142, expected: file:///


```

**Comment by saifjsl** [comment]
Hello @bairathirahul please confirm, this is supposed to be fixed in 1.7.2

Thanks

**Comment by bairathirahul** [comment]
Hi @saifjsl Yes, it was fixed with 1.7.2.
